### DISCUSSION TITLE
What is the name of The Killers' 2004 debut album?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#660] [Galaxy Trivia] [answered]
**Bonus:** How many tracks are on the album?

**Comment by peterbaccaro** [comment, marked as answer]
The Killers' 2004 debut album is called **Hot Fuss**. It includes popular tracks such as "Mr. Brightside," "Somebody Told Me," and "All These Things That I've Done."

The album has a total of **11 tracks**, the tracklist is as follows:

"Jenny Was a Friend of Mine"
"Mr. Brightside"
"Smile Like You Mean It"
"Somebody Told Me"
"All These Things That I've Done"
"Andy, You're a Star"
"On Top"
"Change Your Mind"
"Believe Me Natalie"
"Midnight Show"
"Everything Will Be Alright"
